prow/prowjobs/kubeflow/pipelines/kubeflow-pipelines-postsubmits.yaml

```diff
@@ -5,21 +5,6 @@ postsubmits:
     branches:
     - ^master|release-.+$
     decorate: true
-    env:
-    - name: DOCKER_IN_DOCKER_ENABLED
-      value: "true"
-    volumes:
-    # kubekins-e2e legacy path
-    - name: docker-graph
-      emptyDir: {}
-    # krte (normal) path
-    - name: docker-root
-      emptyDir: {}
-    volumeMounts:
-    - name: docker-graph
-      mountPath: /docker-graph
-    - name: docker-root
-      mountPath: /var/lib/docker
```

Comment:
@Bobgy FYI, these fields were at the wrong indentation level and therefore were not actually applied. I'm removing them to avoid changing the job behavior, but I wanted to let you know in case you'd like to try adding the fields in the correct place. 
Once this PR merges we'll have presubmit validation to catch this type of problem.

Reply:
Thank you for the heads up!